util/cli/recoverer.go

```diff
@@ -0,0 +1,31 @@
+package cli
```

Comment:
`recoverer.go` file name seems quite confusing, why not `run_watcher.go` instead?